src/components/Sum.jsx

```diff
@@ -1 +1,10 @@
-// export function Sum() {}
+/* eslint-disable react/jsx-one-expression-per-line */
+import React from 'react';
+
+export function Sum({ a = 0, b = 0 }) {
+  return (
```

Comment:
let's use arrow syntax
```suggestion
const Sum = ({ a = 0, b = 0 }) => (
```